frontend/packages/common/src/intl/locales/en/system.json

```
@@ -70,7 +70,7 @@
   "messages.recipients-added_other": "{{count}} recipients added to list",
   "messages.recipients-added_one": "{{count}} recipient added to list",
   "messages.cold-chain-no-data-information": "Alert will be triggered if no temperature data is received by the server within the specified timeframe",
-  "messages.cold-chain-temperature-information": "Temperature thresholds are configured in the mSupply Cold Chain application",
+  "messages.cold-chain-temperature-information": "Default Temperature thresholds are 2-8 degrees Celsius for cold chain",
```

It's not clear what 'for cold chain' means here, even in context. Do you mean the cold chain application? 
Regardless, I'm wondering if it matters. It's enough to say that the default is 2 & 8°C
```suggestion

-   "messages.cold-chain-temperature-information": "Default Temperature thresholds are 2°C and 8°C",

```